docs/configuration/prometheus-frontend.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
The idea is that the docs folder still contain valid markdown and it can be useful to download the file if you to give a try. For sure you can still copy/paste the content but it was there already so I thought we might want to keep it if people want quick access to it.

Reply:
Then I guess we should fix `prometheus-frontend.yml` to match `prometheus-frontend.md` content, or am I wrong? The two looks different.

Reply:
Fixed the doc, the yaml change in between, I actually forgot to change it in my refactor but Tom found the problem for his demo.